lib/app/apps.go

```diff
@@ -123,12 +123,12 @@ func StreamAppHookLogs(ctx context.Context, client *kubernetes.Clientset, ref Ho
 }
 
 // DeleteAppHookJob deletes app hook job
-func DeleteAppHookJob(ctx context.Context, client *kubernetes.Clientset, ref HookRef) error {
+func DeleteAppHookJob(ctx context.Context, client *kubernetes.Clientset, req DeleteAppHookJobRequest) error {
 	runner, err := hooks.NewRunner(client)
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	return runner.DeleteJob(ctx, hooks.JobRef{Name: ref.Name, Namespace: ref.Namespace})
+	return runner.DeleteJob(ctx, hooks.JobRef{Name: req.Name, Namespace: req.Namespace}, req.Cascade)
```

Comment:
Have `Runner.DeleteJob` accept an equivalent of `DeleteAppHookJobRequest` instead of dealing with a bunch of attributes (especially a boolean that does not look self-explanatory):
```go
err = runner.DeleteJob(context.TODO(), *ref, false)
```
vs
```go
err = runner.DeleteJob(context.TODO(), DeleteJobRequest{Ref: *ref})
```